docs/how-to/enable_hugepages_for_upf.md

```diff
@@ -0,0 +1,68 @@
+# Enable HugePages for UPF
+
+## Requirements
+
+- A Kubernetes host with a CPU supporting AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+- A Kubernetes cluster with the Multus addon enabled
+- Juju = 3.1/stable
+- HugePages is enabled in Kubernetes Host and it has enough 2Gi free HugePages 
+- Kubernetes Host CPU should have the flag "pdpe1gb" which allows to set 1Gi HugePages
```

Comment:
There are 4 requirements related to Kubernetes I believe we may want to use subbullets:

```
- Juju >= 3.1/stable
- A Kubernetes cluster
  - CPU that supports AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
  - CPU configured with the flag "pdpe1gb" flag
  - Multus Addon enabled
```

Reply:
If we would like to classify all the requirements, the following is more accurate.
- A Kubernetes host 
     - CPU that supports AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
     - CPU has flag "pdpe1gb" which allows to set 1Gi HugePages
     - HugePages is enabled and i has enough 2Gi free HugePages
- A Kubernetes cluster with the Multus addon enabled
-  Juju = 3.1/stable